packages/components/src/Columns/Columns.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { Children } from 'react';
+import { useStyles } from '@marigold/system';
+import { Box } from '../Box';
+import flattenChildren from 'react-keyed-flatten-children';
+
+type ColumnsProps = {
+  className?: string;
+  space?: number;
+  horizontalAlign?: 'left' | 'right' | 'center';
+  verticalAlign?: 'top' | 'bottom' | 'center';
+  title?: string; // Should only be used for testing.
+};
+
+export const Columns: React.FC<ColumnsProps> = ({
+  space = 0,
+  horizontalAlign = 'left',
+  verticalAlign = 'top',
+  className,
+  children,
+  ...props
+}) => {
+  let columnItems = flattenChildren(children);
+  let childClassNames = useStyles({ p: `${space / 2}px` });
```

Comment:
Shouldn't this read from theme?

Reply:
No I think not. It's the padding space we have to pass to the children (Column). User should only pass space to Columns and not to Column, too.

Reply:
Ah, I see `space` is passed from the parent? Does the parent read it from the theme?

Reply:
Our discussion again :D It should read from theme but not at the moment

Reply:
Need a solution here

Reply:
you can pass in a function, e.g.

```
useStyles({ p: theme => theme.spaces[space]/2 });
```

Reply:
not sure about the function signature, could also be an option. autocomplete should work here I guess 🤔